src/core/serializedtypes.js

```diff
@@ -258,8 +258,8 @@ function parse(so) {
 
   const field_bits = tag_byte & 0x0f;
   let field_name = (field_bits === 0)
-    ? field_name = binformat.fields[type_bits][so.read(1)[0]]
```

Comment:
While technically the other syntax was OK, this fix looks much better, it's just that the original style was **so** strange, no?